packages/mui-material/src/TabScrollButton/TabScrollButton.js

```diff
@@ -47,14 +47,41 @@ const TabScrollButtonRoot = styled(ButtonBase, {
 
 const TabScrollButton = React.forwardRef(function TabScrollButton(inProps, ref) {
   const props = useThemeProps({ props: inProps, name: 'MuiTabScrollButton' });
-  const { className, direction, orientation, disabled, ...other } = props;
+  const {
+    className,
+    components = {
+      ScrollButtonStart: KeyboardArrowLeft,
+      ScrollButtonEnd: KeyboardArrowRight,
+    },
```

Comment:
```suggestion
    components = {},
```

The main value with this is that, if developer apply only one, it would crash the component